controllers/profiles/reconciler_dev.go

```diff
@@ -311,11 +317,75 @@ func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *v1.ConfigMap) mutateVis
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = make([]v1.VolumeMount, 0)
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = volumeMounts
 
+			for k, val := range workflow.Labels {
+				if strings.HasPrefix(k, extResPrefix) {
+					cm, err := getGenericConfigMap(val, workflow.Namespace, support)
+					if err != nil {
+						log.Errorf(err, "ConfigMap not found %s", val)
+					} else {
+						deployment := object.(*appsv1.Deployment)
+						for k, _ := range cm.Data {
+							if !isContained(k, deployment.Spec.Template.Spec.Volumes) {
+
+								deployment.Spec.Template.Spec.Volumes = append(deployment.Spec.Template.Spec.Volumes,
+									v1.Volume{
+										Name: configMapExternalPrefix,
```

Comment:
You need to add suffixes accordingly to the resource type. For example, `extres-camel`.

Reply:
ok

Reply:
I think you will have one mount per configMap. For example, a workflow implementing [this example](https://github.com/kiegroup/kogito-examples/tree/main/serverless-workflow-examples/serverless-workflow-camel-routes/src/main/resources):

- `sw.kogito.kie.org/resource-camel`: mount `routes/sendSoapMessage.yaml` (if more routes found in the configMap, more paths. One for each key. Same for the other maps.
- `sw.kogito.kie.org/resource-generic`: mount `generic/numberconversion.wsdl` (here it can be interesting if the CM has a tip for where to mount each key. Maybe the key could contain the file, e.g.: `wsdl/numberconversion.wsdl: <contents>`

The input schema we need to deal with it too. But for that, we have a field in the workflow definition. We can handle it in a different PR/JIRA.

This is a situation where I'm more comfortable having a design doc exposing the use case and all the flows that we might encounter. We've found a few already.

Take a look at the examples we have and try to imagine how we would deploy then in this scenario. We can wrap up the alternative flows for this use case in that document.

Reply:
@ricardozanini 
if the configmap contains two keys 

- camelroute.yaml 
- camelroute.xml 

two volumes and same VolumeMount (src/main/resources/routes) 

missing two mount path in our definitions
camel -> resources/routes
generic-> resources/generic
openapi -> resources/? 
asyncapi-> resources/?